README.md

```diff
@@ -85,9 +85,13 @@ Avaliable Entities and methods
     - `getDistance` - Return the distance specified between two points. The points need to be specified in string format in the format of `POINT(x, y)`
     - `getOverlap` - Determine the overlap between two geometry shapes. Returned as a percentage
     - `runqueue` - Runs the queued up address placement and geometry - address relationship creation jobs stored in the Geometry extension queue. 
+    - `getentity` - Get Entity Geometry relationships, must supply either a `geometry_id` or the combination of entity_id and entity_table
```

Comment:
Rather than use the language "Entity Geometry relationship", I'd prefer simpler language such as "Get relationships between geometries and associated entities"; "Create a relationship between a geometry and an entity"; "Delete a relationship between a geometry and an entity".  Presumably any entity can be specified?  Are there any restrictions on which entity can have a relationship?

Reply:
There are no restrictions on the entity geometry relationship

Reply:
So I think you should document the purpose of the relationship, i.e. what functionality does this module provide when you specify a relationship exists.